string motor_command
uint32 angle
